visidata/macros.py

```diff
@@ -34,16 +35,18 @@ def saveMacro(self, rows, ks):
         macropath = Path(fnSuffix(options.visidata_dir+"macro"))
         vd.save_vd(macropath, vs)
         setMacro(ks, vs)
-        append_tsv_row(vd.macrosheet, (ks, macropath))
+        append_tsv_row(vd.macrosheet.origin, (ks, macropath))
 
 @CommandLog.api
 @wraps(CommandLog.afterExecSheet)
 def afterExecSheet(cmdlog, sheet, escaped, err):
-    cmdlog.afterExecSheet.__wrapped__(cmdlog, sheet, escaped, err)
     if vd.macroMode and (vd.activeCommand is not None) and (vd.activeCommand is not UNLOADED):
         cmd = copy(vd.activeCommand)
         cmd.row = cmd.col = cmd.sheet = ''
         vd.macroMode.addRow(cmd)
+    # happens after, bc
```

Comment:
I didn't understand this comment, what does it mean?

Reply:
This invocation of the original `afterExecSheet` needs to happen after the command is added to the `macro.vd`, because once `afterExecSheet` runs, then `activeCommand` will be set to `None`.

Reply:
I tried to make it clearer